celery/apps/beat.py

```diff
@@ -142,15 +147,15 @@ def startup_info(self, service):
             max_interval=scheduler.max_interval,
         )
 
-    def set_process_title(self):
+    def set_process_title(self) -> None:
         arg_start = 'manage' in sys.argv[0] and 2 or 1
         platforms.set_process_title(
             'celery beat', info=' '.join(sys.argv[arg_start:]),
         )
 
-    def install_sync_handler(self, service):
+    def install_sync_handler(self, service: Service):
         """Install a `SIGTERM` + `SIGINT` handler saving the schedule."""
-        def _sync(signum, frame):
+        def _sync(signum: Signals, frame: FrameType):
```

Comment:
For some reason the pre-commiter prevents me from quoting type hints. Would prefer to do that in order to conditionally import using `if TYPE_CHECKING`. Is this intended?